onnxruntime/core/providers/webnn/builders/impl/resize_op_builder.cc

```diff
@@ -267,15 +267,9 @@ bool ResizeOpBuilder::IsOpSupportedImpl(const InitializedTensorSet& initializers
       return false;
     }
 
-    // coordinate_transformation_mode
-    // Spec issue for supporting more coordinate transformation modes:
-    // https://github.com/webmachinelearning/webnn/issues/270
-    const std::string coordinate_transformation_mode = helper.Get("coordinate_transformation_mode", "half_pixel");
-    if (coordinate_transformation_mode != "half_pixel") {
-      LOGS(logger, VERBOSE) << "Resize does not support coordinate_transformation_mode: "
-                            << coordinate_transformation_mode;
-      return false;
-    }
+    // Ignore coordinate_transformation_mode because WebNN only supports half_pixel mode.
```

Comment:
`half_pixel` is the correct one anyway - nearly all the other modes should be tossed into the recycle bin for bugginess 😉.